optaplanner-wb-webapp/pom.xml

```diff
@@ -825,6 +824,7 @@
     <dependency>
       <groupId>org.jboss.errai</groupId>
       <artifactId>errai-cdi-client</artifactId>
+      <!-- Unlike all other client only dependencies, errai-cdi-client must not be in scope provided -->
```

Comment:
@ge0ffrey the only reason is that these module predate the WB (and naming convention) by quite a bit. errai-cdi-client is needed on the server, same as errai-bus (which is both client and server-side). We could factor out the share classes in errai-cdi-client into a separate module or move them to errai-cdi-server instead. That would be the cleanest solution.